back/src/test/java/com/baba/back/AcceptanceTest.java

```diff
@@ -206,12 +209,27 @@ public class AcceptanceTest {
                 request);
     }
 
-    protected ExtractableResponse<Response> 댓글_삭제_요청(String accessToken, String babyId, Long contentId, Long commentId) {
+    protected ExtractableResponse<Response> 댓글_삭제_요청(String accessToken, String babyId, Long contentId,
+                                                     Long commentId) {
         return delete(String.format("/%s/%s/%s/%s/%s/comment/%s",
                         BASE_PATH, BABY_BASE_PATH, babyId, CONTENT_BASE_PATH, contentId, commentId),
                 Map.of("Authorization", "Bearer " + accessToken));
     }
 
+    protected ExtractableResponse<Response> 성장_앨범_사진_수정_요청(String accessToken, String babyId, Long contentId,
+                                                           UpdateContentPhotoRequest request) throws IOException {
+        final MultipartFile photo = request.photo();
+        return thenExtract(
+                RestAssured.given()
```

Comment:
```suggestion
                RestAssured.given().log().all()
```

성장_앨범_생성_요청도 위처럼 수정해서 request에 대한 로깅도 하는게 어떨까요?

Reply:
네 추가했습니다!